libraries/AP_Scripting/examples/mission-offset.lua

```diff
@@ -0,0 +1,91 @@
+-- This script offsets the created mission to vehicle's current location
+-- Required setup RCx_OPTION = 300 (Scripting1)
+-- When this switch is pulled high, move the entire waypoint so that the first waypoint is at the current position
+-- Waypoints with lan=0 and lng=0 are ignored
+
+local current_pos -- current position of the vehicle
+local rc_option = 300 -- Scripting1
+local sw_mission_offset_pos_prev = -1 -- previous position of the switch
+
+function ahrs_ready()
+  current_pos = ahrs:get_position()
+  if current_pos == nil then
```

Comment:
you can also do  
```suggestion
  if not current_pos then
```
Not that is makes any difrence to the function.

Reply:
I changed it.